src/components/input/input.test.jsx

```diff
@@ -296,4 +297,20 @@ describe('Input', () => {
       expect(tree.findByType('input').props.value).toEqual('test-value');
     });
   });
+
+  describe('ref API', () => {
+    it('is set to the inputRef property', () => {
```

Comment:
This is just a note for future devs: we're moving away from `react-test-renderer` and `enzyme`. How would we test this with `@testing-library/react`?

Reply:
React testing-library uses deep rendering by default, so no special additions would be required to check/test the prop values.